var/spack/repos/builtin/packages/llvm-openmp-ompt/package.py

```diff
@@ -38,7 +38,7 @@ class LlvmOpenmpOmpt(CMakePackage):
     depends_on('llvm', when='~standalone')
     depends_on('ninja@1.5:', type='build')
     depends_on('perl@5.22.0:', type='build')
-    depends_on('libelf', when='+libomptarget')
+    depends_on('elf', when='+libomptarget')
```

Comment:
I want to check: you replace a dependency on `elf` with `elfutils` in several packages (e.g. in `openspeedshop-utils`) - is that required?

Generally packages should explain if they require a specific implementation of a virtual. And likewise we should be careful about replacing a specific request (for `libelf`) with a more-general one (for `elf`). That being said I'm looking through the history of this and the `llvm` package and not seeing a comment on why this specific implementation is needed.

If you are building a spec with this package and using `elfutils`, then I think that is sufficient information to relax the dependency here.

Reply:
@scheibelp Yes, we are building with elfutils.

Reply:
@scheibelp @lee218llnl Hi Peter, Greg, Did you see anything else that needs changing?   Thanks for your help!